### ISSUE TITLE
fido-authenticator: “up needed” not signaled on NK3AM

### ISSUE THREAD

**Issue by robin-nitrokey** [Nitrokey/nitrokey-3-firmware#96]
> one curious thing is that I get the prompt to touch the device in the 3A NFC but it doesn't prompt with 3A mini
> It seems if status == STATUS.UPNEEDED: # Waiting for touch this call in python-fido2 isn't called on the 3aMini

https://github.com/Nitrokey/pynitrokey/pull/274

**Comment by runcom** [comment]
I also have the feeling that this https://github.com/Nitrokey/nitrokey-3-firmware/issues/94 is also related to the issue here and the firmware is _missing something_ - although I have no way to test what's going on at the hardware/firmware level

**Comment by robin-nitrokey** [comment]
In combination with https://github.com/Nitrokey/nitrokey-3-firmware/issues/93, my guess that there is something wrong with the timers that trigger the `ctaphid_keepalive` and `ui` tasks:  the `CTAPHID_KEEPALIVE` command signals `STATUS_UPNEEDED`, and `ui` should call the code that toggles the LED if it is blinking.  @daringer do you have an idea why this could be the case?  I will also try to debug this while working on https://github.com/Nitrokey/nitrokey-3-firmware/issues/79.

**Comment by daringer** [comment]
looks like this is also fixed by #319 

based on the original issue (https://github.com/Nitrokey/pynitrokey/pull/274) I tested using `nitropy fido2 verify` and both behave identical now:
* nk3am:
```
❯ nitropy fido2 verify
Command line tool to interact with Nitrokey devices 0.4.38
Enter PIN: 
Touch your authenticator device now...
/../
```
* nk3an:
```
❯ nitropy fido2 verify
Command line tool to interact with Nitrokey devices 0.4.38
Enter PIN: 
Touch your authenticator device now...
/../
```

so, should we close this as fixed?

**Comment by robin-nitrokey** [comment]
AFAIS this is fixed in current firmware releases, so I’m closing this issue.  Please re-open if the problem occurs again.
